components/input/TextArea.razor.cs

```diff
@@ -32,56 +32,85 @@ public partial class TextArea : Input<string>
         [Parameter]
         public bool AutoSize { get; set; }
 
+        /// <summary>
+        /// When `false`, value will be set to `null` when content is empty 
+        /// or whitespace. When `true`, value will be set to empty string.        
+        /// </summary>
         [Parameter]
         public bool DefaultToEmptyString { get; set; }
 
+        /// <summary>
+        /// `TextArea` will allow growing, but it will stop when visible 
+        /// rows = MaxRows (will not grow further).
+        /// Default value = uint.MaxValue
+        /// </summary>
         [Parameter]
-        public uint MinRows
+        public uint MaxRows
         {
             get
             {
-                return _minRows;
+                return _maxRows;
             }
             set
             {
                 _hasMinOrMaxSet = true;
-                if (value >= DEFAULT_MIN_ROWS && value <= MaxRows)
+                if (value >= MinRows)
                 {
-                    _minRows = value;
+                    _maxRows = value;
                     AutoSize = true;
                 }
                 else
                 {
-                    throw new ArgumentOutOfRangeException(nameof(MinRows), $"Please enter a value between {DEFAULT_MIN_ROWS} and {MaxRows}");
+                    throw new ArgumentOutOfRangeException($"Please enter a value between {MinRows} and {uint.MaxValue}");
                 }
             }
         }
 
+        /// <summary>
+        /// `TextArea` will allow shrinking, but it will stop when visible 
+        /// rows = MinRows (will not shrink further).
+        /// Default value = DEFAULT_MIN_ROWS = 1
+        /// </summary>
         [Parameter]
-        public uint MaxRows
+        public uint MinRows
         {
             get
             {
-                return _maxRows;
+                return _minRows;
             }
             set
             {
                 _hasMinOrMaxSet = true;
-                if (value >= MinRows)
+                if (value >= DEFAULT_MIN_ROWS && value <= MaxRows)
                 {
-                    _maxRows = value;
+                    _minRows = value;
                     AutoSize = true;
                 }
                 else
                 {
-                    throw new ArgumentOutOfRangeException($"Please enter a value between {MinRows} and {uint.MaxValue}");
+                    throw new ArgumentOutOfRangeException(nameof(MinRows), $"Please enter a value between {DEFAULT_MIN_ROWS} and {MaxRows}");
```

Comment:
Can we fallback to the minimum or maximum value?

Reply:
Sure. Probably safer to fall back to minimum? Also, maybe we do a `Console.Debug` here then?

Reply:
That's fine. And the maximum for `MaxRows`.